src/app/[locale]/join-us/__tests__/__snapshots__/page.test.tsx.snap

```diff
@@ -59,19 +59,19 @@ exports[`Support Us page > renders correctly 1`] = `
             <div
               class="text-rose-600 text-base font-bold leading-normal"
             >
-              Support Us
+              Join Us
             </div>
           </div>
           <div
             class="self-stretch text-center text-white text-[56px] lg:text-[80px] leading-[114.286%] md:leading-[120%] font-semibold"
           >
-            What we’re trying to achieve
+            Join us in shaping the future of open source Java
```

Comment:
```suggestion
            Join us in ensuring the future of open source Java
```

I realise this is a small change, but Adoptium is does not define/shape Java itself - there are other processes to do that. This change demonstrates that we are doing our part to ensure open Java remains available for everyone.